### DISCUSSION TITLE
Input from users.

### DISCUSSION THREAD

**Discussion by parasnayakjain** [vedicscript/vedicscript#5] [General]
Hi, Paras Here
Can we get values from users and initialise them?

**Comment by PtPrashantTripathi** [comment]
> Hi, Paras Here Can we get values from users and initialise them?

@parasnayakjain Nice idea, we can include this in new relese
Thanks for suggestion

**Comment by parasnayakjain** [comment]
Thanks for your reply.
What is your motto behind this project?
Is it just for the promotion of Sanskrit?
Or some technical goals.

**Reply by PtPrashantTripathi** [reply]
I don't think Sanskrit need any promotion,
I have just created it out of curiosity 💚

**Reply by parasnayakjain** [reply]
As Sanskrit is a very powerful language. The order of words doesn't make a change in the meaning of the sentence and everything is defined by strict laws of Sanskrit grammar. It is well known it has huge potential in the computer science field.
But that potential has never been utilized (in modern times). 
You can scale up your project in this direction.

**Comment by parasnayakjain** [comment]
Can I know more about you?

**Reply by PtPrashantTripathi** [reply]
https://ptprashanttripathi.github.io
